osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/endpoints/SmartMeteringMonitoringEndpoint.java

```diff
@@ -512,12 +516,70 @@ public ClearAlarmRegisterResponse getClearAlarmRegisterResponse(
         } catch (final FunctionalException e) {
             throw e;
         } catch (final Exception e) {
-            LOGGER.error("Exception: {} while sending clear alarm register request of device: {} for organisation {}.",
+            log.error("Exception: {} while sending clear alarm register request of device: {} for organisation {}.",
                     e.getMessage(), request.getDeviceIdentification(), organisationIdentification);
 
             this.handleException(e);
         }
         return response;
     }
+    
+    @PayloadRoot(localPart = "ActualPowerQualityRequest", namespace = SMARTMETER_MONITORING_NAMESPACE)
+    @ResponsePayload
+    public ActualPowerQualityAsyncResponse getActualPowerQuality(
+            @OrganisationIdentification final String organisationIdentification,
+            @RequestPayload final org.opensmartgridplatform.adapter.ws.schema.smartmetering.monitoring.ActualPowerQualityRequest request, @MessagePriority final String messagePriority,
+            @ResponseUrl final String responseUrl, @ScheduleTime final String scheduleTime) throws OsgpException {
+
+        log.debug("Incoming ActualPowerQualityRequest for meter: {}.", request.getDeviceIdentification());
+
+
```

Comment:
Use at most a single blank line to create visual separation.

Reply:
Changed